org.scala-ide.sdt.debug.expression/src/org/scalaide/debug/internal/expression/proxies/JdiProxy.scala

```diff
@@ -95,10 +93,10 @@ trait JdiProxy extends Dynamic {
 /**
  * Base for companion objects for [[org.scalaide.debug.internal.expression.proxies.JdiProxy]].
  */
-trait JdiProxyCompanion[Proxy <: JdiProxy, Underlying <: Value] {
+trait JdiProxyCompanion[P <: JdiProxy, Underlying <: Value] {
```

Comment:
Ok, there is a `scala.Proxy` in every scope, meh. I would either use `import scala.{ Proxy => _ }` or rename it to `ProxyType`.